### DISCUSSION TITLE
Copy settings across user accounts (e.g. Firefox setting)

### DISCUSSION THREAD

**Discussion by tichzvidzayi** [ltsp/ltsp#381] [Q&A]
Out of curiosity is there a way to copy e.g. Firefox browser settings across multiple user accounts without doing it manually for each user.

**Comment by maccadia** [comment]
Hi, for Firefox, have a look at https://support.mozilla.org/en-US/kb/customizing-firefox-using-policiesjson.

For Gnome, https://help.gnome.org/admin/system-admin-guide/stable/.

You can also use `/etc/skel` to define default settings for new users.

**Reply by tichzvidzayi** [reply]
thanks
